maxim/logger/livekit/module_functions.py

```
@@ -0,0 +1,116 @@
+                    "id": fn_call.id,
+                    "name": fn_call.name,
+                    "description": tool_call.__livekit_tool_info.description
+                    if tool_call.__livekit_tool_info.description is not None
+                    else "",
+                    "args": fn_call.args,
+                    "start_timestamp": start_timestamp,
+                }
+            )
+            tool_call.result(output.output)
+
+    except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)